PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs

```diff
@@ -88,6 +95,7 @@ private void Init()
         {
             try
             {
+                InitSizePosition();
```

Comment:
move this call before line `Logger.Log("PSL begins");`, and you will feel the effect better.
reason: see `SetTimeout(Init, 800);`

I'm fine with the rest of codes